tests/test_modeling_blenderbot.py

```diff
@@ -0,0 +1,213 @@
+import unittest
+import tempfile
+from transformers import is_torch_available
+
+if is_torch_available():
+    import torch
+    from transformers import (
+        AutoModel,
+        AutoTokenizer,
+        BlenderbotTokenizer,
+        BlenderbotConfig,
+        BlenderbotConditionalGeneration,
+        BLENDERBOT_PRETRAINED_MODEL_ARCHIVE_LIST
+    )
+from .utils import require_torch, slow, torch_device
+from .test_modeling_common import ids_tensor, ModelTesterMixin
+from .test_configuration_common import ConfigTester
+from transformers.modeling_bart import _prepare_bart_decoder_inputs
+
+weights_path = "/home/mariama/PycharmProjects/blender/blender_90M"  # won't be needed  once the pretrained weights uploaded on S3
+
+@require_torch
+class BlenderbotModelTester:
+    def __init__(self,
+                 parent,
+                 batch_size=4,
+                 seq_len=17,
+                 vocab_size=100,
+                 hidden_size=16,
+                 is_training=False,
+                 num_hidden_layers=2,
+                 num_attention_heads=2,
+                 hidden_dropout_prob=0.2,
+                 max_position_embeddings=50,
+                 eos_token_id=2,
+                 bos_token_id=1,
+                 pad_token_id=0,
+                 use_labels=True,
+                 ffn_size=4,
+                 attention_dropout=0.2,
```

Comment:
(nit) you can make this smaller by just writing 
```python
self.seq_len = 17
```
Instead of having a kwarg that is never used.

Reply:
Agree with @sshleifer here